recipes/einsteinpy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "einsteinpy" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 8a8815ae9341f4b544c496a697342892
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True
+
+requirements:
+  build:
+    - python
+    - setuptools
+  
+  host:
+    - pip
+    - python
+
+  run:
+    - python
+    - numpy >= 1.14
+    - astropy
+    - matplotlib
+    - scipy
+
+test:
+  imports:
+    - {{ name }}
+    - {{ name }}.integrators
+    - {{ name }}.metric
+    - {{ name }}.plotting
+    - {{ name }}.utils
+
+  requires:
+    - pytest
+    - pytest-benchmark
+
+about:
+  home: https://{{ name }}.github.io/
+  license: MIT
+  summary: "Python Library for General Relativity!"
+  doc_url: https://{{ name }}-project.readthedocs.io/
+  dev_url: https://github.com/{{ name }}/{{ name }}/blob/master/docs/source/user_guide.rst
+
+extra:
+  recipe-maintainers:
+    - shreyasbapat
+    - ritzvik
```

Comment:
@ritzvik @spino17 Are you happy to be listed as maintainers of this recipe?

Reply:
@chrisburr 👍 from my side
@spino17 also wouldn't mind